lib/obsolete.gd

```diff
@@ -760,3 +760,8 @@ BindGlobal("STRING_LIST_DIR", function(dirname)
       return JoinStringsWithSeparator( list, "\000" );
     fi;
 end);
+
+
+DeclareObsoleteSynonym("GAP_EXIT_CODE", "GapExitCode", 2);
```

Comment:
Please also add a comment here that explains why these are here, like we have for everything else in here. It really helps to have these.